projects/packages/connection/src/class-urls.php

```diff
@@ -82,8 +82,8 @@ public static function normalize_www_in_url( $option, $url_function ) {
 	/**
 	 * Return URL with a normalized protocol.
 	 *
-	 * @param callable $callable Function to retrieve URL option.
-	 * @param string   $new_value URL Protocol to set URLs to.
+	 * @param string $callable Function that was used to retrieve URL option.
```

Comment:
Non-blocker, but a bit more clear:

```suggestion
	 * @param string $callable Function name that was used to retrieve URL option.
```